src/game/frontend/submenus/Self.cpp

```diff
@@ -44,11 +54,13 @@ namespace YimMenu::Submenus
 	Self::Self() :
 	    Submenu::Submenu("Self")
 	{
-		auto main          = std::make_shared<Category>("Main");
-		auto globalsGroup  = std::make_shared<Group>("Globals", GetListBoxDimensions());
-		auto movementGroup = std::make_shared<Group>("Movement", GetListBoxDimensions());
-		auto toolsGroup    = std::make_shared<Group>("Tools", GetListBoxDimensions());
-		auto columns       = std::make_shared<Column>(2);
+		auto main              = std::make_shared<Category>("Main");
+		auto globalsGroup      = std::make_shared<Group>("Globals", GetListBoxDimensions());
+		auto movementGroup     = std::make_shared<Group>("Movement", GetListBoxDimensions());
+		auto toolsGroup        = std::make_shared<Group>("Tools", GetListBoxDimensions());
+		auto collectiblesGroup = std::make_shared<Group>("Collectibles", GetListBoxDimensions());
```

Comment:
Since recovery can have many parts such as roles and boosting, could you seclude it to its own sub-menu? Either within the Self menu or completely its own. Both would be okay.